frontend/src/features/videos/VideoCard.tsx

```diff
@@ -264,12 +238,18 @@ function VideoCard({
   }
 
   return (
-    <Grid container spacing={1} className={classes.main}>
+    <Grid
+      container
+      spacing={1}
+      sx={{
+        ...mainSx(theme),
+      }}
```

Comment:
`sx` accepts a callback, and other "theme-aware" properties: https://mui.com/system/the-sx-prop/#theme-aware-properties
```suggestion
    <Grid container spacing={1} sx={mainSx}>
```

Reply:
@amatissart Thanks for letting me know, I just updated the two components (AccountInfo and VideoCard) in which we are using Theme Object directly.